hypervisor/arch/x86/mmu.c

```diff
@@ -247,6 +250,9 @@ void init_paging(void)
 
 	pr_dbg("HV MMU Initialization");
 
+	/* Initialize IA32_PAT according to ISDM 11.12.4 Programming the PAT */
+	msr_write(MSR_IA32_PAT, PAT_POWER_ON_VALUE);
+
```

Comment:
init_paging will eventually call enable_paging. So we don't need the changes here.

Reply:
> init_paging will eventually call enable_paging. So we don't need the changes here.

Yes, it's embarrassing ... will fix.